### PR TITLE
MAR-4055-buyer-frontend-change-brief-to-opportunity

### PR DESCRIPTION
**Pull request by Revit**

### CODE REVIEW

**Review by Reshma-M-A — COMMENTED** [review]
Lets revert the changes for the following files:

- app/templates/content/terms-of-use/2016-09-02 11:00.html
- app/templates/content/terms-of-use/2017-02-22 12:00.html
- app/templates/content/privacy-policy.html
for legal purposes

**Review by Reshma-M-A — COMMENTED** [review]
Do we need to edit the file `test_build_brief.py` and `helpers.py` (/tests/integration/bdd/helpers.py) ?

**Review by tobiasbrown — COMMENTED** [review]
> Do we need to edit the file `test_build_brief.py` and `helpers.py` (/tests/integration/bdd/helpers.py) ?

No, I don't think we need to worry about updating these files.

File: app/templates/buyers/edit_brief_question.html [hunk 1]
```diff
@@ -14,7 +14,7 @@
       },
       {
         "link": '/2/brief/%s/overview' % brief.id if brief.lot == 'digital-professionals' or brief.lot == 'training' else url_for(".view_brief_overview", framework_slug=brief.frameworkSlug, lot_slug=brief.lotSlug, brief_id=brief.id),
-        "label": "Brief Overview"
+        "label": "Opportunity Overview"
```
Comment (tobiasbrown) [inline app/templates/buyers/edit_brief_question.html:17]: There are occurrences in:
- brief_publish_confirmation.html
- brief_responses.html
- dashboard.html

File: tests/app/views/test_buyers.py [hunk 2]
```diff
@@ -1403,7 +1405,20 @@ def test_show_draft_brief_summary_page(self, data_api_client):
...
+                    'Specialist role',
+                    'Location',
+                    'Description of work',
+                    'Shortlist and evaluation process',
+                    'How long your brief will be open',
+                    'Question and answer session details',
+                    'Review and publish your requirements'
+                ]
+            """
+            assert [e.text_content() for e in document.xpath('//*[@id="content"]//ul[@class="instruction-list-item-top"]/li/a')] in [  # noqa
```
Comment (Reshma-M-A) [inline tests/app/views/test_buyers.py:1421]: To remove the commented code

File: app/templates/brief.html [hunk 3]
```diff
@@ -44,17 +44,17 @@ <h2>
...
         <p class="banner-message">{% if brief_of_current_user %}
-          <p>This brief is not yet published.</p> {% if unanswered_required > 0 %}
-          <p>Edit the brief to complete all mandatory questions.</p>
+          <p>This opportunity is not yet published.</p> {% if unanswered_required > 0 %}
+          <p>Edit the opportunity to complete all mandatory questions.</p>
             {% if brief.lot == 'digital-outcome' %}
               <a href={{ url_for('buyers.view_brief_overview', framework_slug=brief.frameworkSlug, lot_slug=brief.lotSlug, brief_id=brief.id) }} class="button">Edit Brief</a>
             {% else %}
-              <a href="{{ '/2/brief/%s/overview' % brief.id }}" class="button">Edit Brief</a>
+              <a href="{{ '/2/brief/%s/overview' % brief.id }}" class="button">Edit Opportunity</a>
```
[the remark this answers is not in the document]
Comment (Revit) [reply app/templates/brief.html]: changed as suggested

File: app/templates/buyers/edit_brief_question.html (hunk 1 above)
Comment (Revit) [reply app/templates/buyers/edit_brief_question.html:17]: changed as suggested

Comment (Revit) [reply app/templates/buyers/edit_brief_question.html:17]: changes as suggested

File: tests/app/views/test_buyers.py [hunk 4]
```diff
@@ -28,6 +28,8 @@
 Nice2,False,True,True,True,False
 Nice3,False,True,False,True,False"""
 
+""" Test Buyer """
```
[the remark this answers is not in the document]
Comment (Revit) [reply tests/app/views/test_buyers.py:31]: Removed comments

File: tests/app/views/test_buyers.py [hunk 5]
```diff
@@ -1403,16 +1403,18 @@ def test_show_draft_brief_summary_page(self, data_api_client):
...
+                assert e.text_content() in [
+                    'Title',
+                    'Specialist role',
+                    'Location',
+                    'Description of work',
+                    'Shortlist and evaluation process',
+                    'How long your brief will be open',
+                    'Question and answer session details',
+                    'Review and publish your requirements'
+                ]
```
Comment (tobiasbrown) [inline tests/app/views/test_buyers.py:1417]: This change looks unrelated.  Is there a reason why this needed to be updated?

Comment (tobiasbrown) [reply tests/app/views/test_buyers.py:1417]: Also check app/templates/brief.html for `Brief`.  Let's change that to `opportunity`.

Comment (Revit) [reply tests/app/views/test_buyers.py:1417]: The reason I added the new codes from 1406 to 1417 line is because the existing codes failed on CycleCI and local. New codes serve the same purpose as previous except passed in build

File: app/templates/brief.html [hunk 6]
```diff
@@ -44,17 +44,17 @@ <h2>
           Preview only
         </h2>
         <p class="banner-message">{% if brief_of_current_user %}
-          <p>This brief is not yet published.</p> {% if unanswered_required > 0 %}
-          <p>Edit the brief to complete all mandatory questions.</p>
+          <p>This opportunity is not yet published.</p> {% if unanswered_required > 0 %}
+          <p>Edit the opportunity to complete all mandatory questions.</p>
             {% if brief.lot == 'digital-outcome' %}
-              <a href={{ url_for('buyers.view_brief_overview', framework_slug=brief.frameworkSlug, lot_slug=brief.lotSlug, brief_id=brief.id) }} class="button">Edit Brief</a>
+              <a href={{ url_for('buyers.view_brief_overview', framework_slug=brief.frameworkSlug, lot_slug=brief.lotSlug, brief_id=brief.id) }} class="button">Edit Opportunity</a>
```
Comment (tobiasbrown) [inline app/templates/brief.html:50]:
```suggestion
              <a href={{ url_for('buyers.view_brief_overview', framework_slug=brief.frameworkSlug, lot_slug=brief.lotSlug, brief_id=brief.id) }} class="button">Edit opportunity</a>
```

**Review by tobiasbrown — APPROVED** [review]
